homework/index.js

```diff
@@ -29,18 +29,78 @@
     });
     return elem;
   }
+  function renderRepositories(repository, container) {
```

Comment:
The naming of these functions is 👍! It's very "React" to call this `render...`.

However a small thing here: this function renders *one* repository, so it should also be named `renderRepository`.